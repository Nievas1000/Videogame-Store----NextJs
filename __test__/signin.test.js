import { render } from "@testing-library/react";
import SignIn from "../pages/signin";

describe("SignIn", () =>{

    it("render SignIn", () => {
        const {sigin} = render(<SignIn />)
        expect(sigin).toMatchSnapshot()
    })
})
