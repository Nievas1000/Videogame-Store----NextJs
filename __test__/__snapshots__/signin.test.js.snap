// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn render SignIn 1`] = `undefined`;
